test/testlib/nuodb_admin_utilities.go

```diff
@@ -142,6 +143,18 @@ func StartAdminTemplate(t *testing.T, options *helm.Options, replicaCount int, n
 		}
 	}
 
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+
+	RunOnNuoDBVersionCondition(t, ">=6.0.0", func(version *semver.Version) {
+		//With NuoDB v6.0.0 a license is mandatory to add
+		if os.Getenv("NUODB_LICENSE_CONTENT") == "" {
+			t.Error("License required to start NuoDB tests.")
+		}
+		ApplyLicense(t, namespaceName, admin0, ENTERPRISE)
```

Comment:
You have added another env variable that has the limited license `NUODB_LIMITED_LICENSE_CONTENT`. I think it makes more sense to install it for all the tests now. E.g.

```go
if os.Getenv("NUODB_LIMITED_LICENSE_CONTENT") == "" {
  t.Error("Limited license required to start NuoDB tests.")
}
ApplyLicense(t, namespaceName, adminNames[0], LIMITED)
```